src/applications/gi/components/Dropdown.jsx

```diff
@@ -30,7 +23,7 @@ class Dropdown extends React.Component {
           alt={this.props.alt}
           value={this.props.value}
           onChange={this.props.onChange}
-          onFocus={this.handleFocus}
+          onFocus={this.props.onFocus.bind(this, this.dropdownId)}
```

Comment:
```suggestion
          onFocus={environment.isProduction() ? this.handleFocus : () => this.props.onFocus(this.dropdownId)}
```

Reply:
I don't think we are going to get this to work without binding. 
https://reactjs.org/docs/handling-events.html